style.css

```diff
@@ -260,13 +250,131 @@
     flex-grow: 0;
   }
 
+  .project-modal {
+    display: none;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    position: fixed;
+    z-index: 1;
+    padding-top: 40px;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  .project-action-buttons {
+    display: flex;
+    padding: 4px 9px;
+    justify-content: space-between;
+    align-items: center;
+    gap: 12px;
+    color: #6070ff;
+    width: 95%;
+  }
+
+  .view-project-btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    border-radius: 8px;
+    padding: 5px;
+    border: 1px solid #6070ff;
+    width: 136px;
+    height: 48px;
+  }
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement as I mentioned in my previous comments because I can see you are not using the padding or spacing correctly. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-28-46](https://github.com/NoelLincoln/mportfolio/assets/66476329/482c01ae-ee57-41ad-b39e-ec62edb55125)

</td>
<td>

![Screenshot from 2023-06-30 21-29-04](https://github.com/NoelLincoln/mportfolio/assets/66476329/bfd70ea0-a1f0-4836-94f9-e066f305a691)

</td>
</tr>
</table>